.github/dependabot.yml

```diff
@@ -1,12 +1,26 @@
 version: 2
 
 updates:
-  - package-ecosystem: cargo
-    directory: /
+  - package-ecosystem: "github-actions"
+    directory: "/"
     schedule:
-      interval: daily
+      interval: "weekly"
 
-  - package-ecosystem: github-actions
-    directory: /
+  - package-ecosystem: "cargo"
+    directory: "/"
     schedule:
-      interval: daily
+      interval: "weekly"
+    groups:
+      wgpu-dependencies:
+        patterns:
+        - "futures-intrusive"
+        - "pollster"
+        - "text_placeholder"
+        - "wgpu"
+      wasm-bindgen-dependencies:
+        patterns:
+        - "js-sys"
+        - "wasm-bindgen"
+        - "wasm-bindgen-futures"
+        - "wasm-logger"
+        - "wasm-timer"
```

Comment:
I would add a comment on top of each group to describe why they are required.

Reply:
Yep, I agree. I haven't considered other groups because I do not know which could be. If you someone has other dependencies groups in mind, I can add them to this PR, otherwise we can add further ones later

Reply:
Ok, added comments. We can add more groups later, so we can merge this PR in my opinion